ui/src/timeMachine/actions/queries.ts

```diff
@@ -131,8 +131,11 @@ export const executeQueries = (dashboardID?: string) => async (
     dispatch(setQueryResults(RemoteDataState.Loading, [], null))
 
     await dispatch(refreshTimeMachineVariableValues(dashboardID))
```

Comment:
this updates the state. So when we we were using `state` beforehand, the `state` we had referenced was changed here, but those changes weren't reflected on the `state` variable